src/EFCore.PG/Query/ExpressionTranslators/Internal/NpgsqlStringMethodTranslator.cs

```diff
@@ -425,6 +431,30 @@ public NpgsqlStringMethodTranslator(NpgsqlTypeMappingSource typeMappingSource, I
                 _typeMappingSource.FindMapping(typeof(string[])));
         }
 
+        if (method == ToDate)
+        {
+            return _sqlExpressionFactory.Function(
+                "to_date",
+                new[] { arguments[1], arguments[2] },
+                nullable: true,
+                argumentsPropagateNullability: new[] { true, false },
```

Comment:
If the 2nd argument (the format) is nullable, doesn't to_date return null? That's what the `false` here seems to be implying.

Reply:
Yeah, that was actually a parameter that I still find a bit weird to understand, but after reading your comment https://github.com/dotnet/efcore/issues/23042#issuecomment-711410958 I will change that as well.